src/modules/src/controller.c

```diff
@@ -25,7 +25,7 @@ typedef struct {
 static ControllerFcns controllerFunctions[] = {
   {.init = 0, .test = 0, .update = 0, .name = "None"}, // Any
   {.init = controllerPidInit, .test = controllerPidTest, .update = controllerPid, .name = "PID"},
-  {.init = controllerMellingerInit, .test = controllerMellingerTest, .update = controllerMellinger, .name = "Mellinger"},
+  {.init = controllerMellingerFirmwareInit, .test = controllerMellingerFirmwareTest, .update = controllerMellingerFirmware, .name = "Mellinger"},
```

Comment:
For the sake of consistency, is it perhaps better to switch the names of these functions? Call these controllerMellinger* like with the others and those that take the controller struct  controllerMellingerBase* or something?

Reply:
The goal was to keep using the "controllerMellinger*" name for the "public" functions that also end up in the bindings. Ultimately, this sort of structural change will need to be done for all controllers that we want to have bindings for.

Reply:
sounds fine by me!